### PR TITLE
Ability to use CDN URL in .npmrc file

### PR DESCRIPTION
**Pull request by wynnfarm**
Enhancements/Changes made -
- I added the ability to implement the CDN URL inside the .npmrc file similarly to how the chromedriver allows it.  
- Updated some of the console logs to be more descriptive during install.  
- Updated the README to provide an explanation of how the CDN URL can be used in the .npmrc file along with the default value if no CDN URL is provided.

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -11,16 +11,17 @@ var Promise = require('bluebird');
...
-var DOWNLOAD_MAC = baseCDNURL + '/v0.19.0/geckodriver-v0.19.0-macos.tar.gz';
-var DOWNLOAD_LINUX64 = baseCDNURL + '/v0.19.0/geckodriver-v0.19.0-linux64.tar.gz';
-var DOWNLOAD_LINUX32 = baseCDNURL + '/v0.19.0/geckodriver-v0.19.0-linux32.tar.gz';
-var DOWNLOAD_WIN32 = baseCDNURL + '/v0.19.0/geckodriver-v0.19.0-win32.zip';
-var DOWNLOAD_WIN64 = baseCDNURL + '/v0.19.0/geckodriver-v0.19.0-win64.zip';
+  var DOWNLOAD_MAC = baseCDNURL + '/v0.19.0/geckodriver-v0.19.0-macos.tar.gz';
+  var DOWNLOAD_LINUX64 = baseCDNURL + '/v0.19.0/geckodriver-v0.19.0-linux64.tar.gz';
+  var DOWNLOAD_LINUX32 = baseCDNURL + '/v0.19.0/geckodriver-v0.19.0-linux32.tar.gz';
+  var DOWNLOAD_WIN32 = baseCDNURL + '/v0.19.0/geckodriver-v0.19.0-win32.zip';
+  var DOWNLOAD_WIN64 = baseCDNURL + '/v0.19.0/geckodriver-v0.19.0-win64.zip';
```
Comment (vladikoff) [inline index.js]: What happened here with the spacing?

File: index.js [hunk 2]
```diff
@@ -38,11 +39,11 @@ if (platform === 'win32') {
   executable = 'geckodriver.exe';
 }
 
-process.stdout.write('Downloading geckodriver... ');
+process.stdout.write(`Downloading geckodriver from ${downloadUrl}...\n`);
```
Comment (vladikoff) [inline index.js]: this breaks tests, https://github.com/vladikoff/node-geckodriver/blob/master/test/index.js#L9

**Comment by wynnfarm** [comment]
The automated tests that are ran after each update appear to be running with code not in the most current commit.  I reverted all test code and the console log statements that should be creating the log statement below but for some reason when the automated test runs it still produces that code in the console.  Any idea as to why that is?

"Downloading geckodriver from https://github.com/mozilla/geckodriver/releases/download/v0.19.0/geckodriver-v0.19.0-win64.zip...\nExtracting geckodriver.zip to C:\\projects\\node-geckodriver...\nComplete.\n"

**Comment by vladikoff** [comment]
landed in https://github.com/vladikoff/node-geckodriver/pull/26

**Pull request closed without merging**